docs/operator-manual/webhook.md

```diff
@@ -27,11 +27,13 @@ accessible, then configuring a webhook secret is recommended to prevent a DDoS a
 In the `argocd-secret` kubernetes secret, configure one of the following keys with the Git
 provider's webhook secret configured in step 1.
 
-| Provider  | K8s Secret Key           |
-|---------- | ------------------------ |
-| GitHub    | `github.webhook.secret`  |
-| GitLab    | `gitlab.webhook.secret`  |
-| BitBucket | `bitbucket.webhook.uuid` |
+| Provider        | K8s Secret Key                   |
+|-----------------| ---------------------------------|
+| GitHub          | `webhook.github.secret`          |
+| GitLab          | `webhook.gitlab.secret`          |
+| BitBucket       | `webhook.bitbucket.uuid`         |
+| BitBucketServer | `webhook.bitbucketserver.secret` |
```

Comment:
do we want to point out which one is which for new users?

Reply:
The doc was using a table to list the supported providers and secret options. I added the two know ones and fix the typo on the 3 previous ones.